addon/components/map-commands-dialogs/go-to.js

```diff
@@ -315,6 +315,17 @@ let FlexberryGoToMapCommandDialogComponent = Ember.Component.extend({
 
         this.set('_options.x', x);
         this.set('_options.y', y);
+      },
+
+      /**
+        Handles coordinate input textboxes keyPress events.
+
+        @method actions.coordsInputKeyPress
+      */
+      coordsInputKeyPress(e) {
+        if (e.charCode !== 45 && e.charCode !== 44 && e.charCode !== 46 && e.charCode && (e.charCode < 48 || e.charCode > 57)) {
```

Comment:
Use of `e.charCode` isn't a cross-browser variant, use [jQuery's `e.which`](http://api.jquery.com/event.which/) instead.
If `e` is native browser event, then it can be transformed into jQuery event with call to `Ember.$.event.fix(e)`.